AutomationTestingProject/src/test/java/com/evozon/BaseTest.java

```diff
@@ -0,0 +1,7 @@
+package com.evozon;
+
+public class BaseTest {
+    public void beforeEach() {
```

Comment:
Puteai declara si instantia webdriver-ul si aici, doar, nu in fiecare @Before din clasele child